toolkits/golang_test.go

```diff
@@ -200,3 +201,65 @@ func Test_goBuildStep(t *testing.T) {
 		})
 	}
 }
+
+func Benchmark_goBuildStep(b *testing.B) {
+	isInstallRequired, _, goConfig, err := selectGoConfiguration()
+	require.NoError(b, err, "Failed to select an appropriate Go installation for compiling the Step")
+
+	if isInstallRequired {
+		b.Fatalf("Failed to select an appropriate Go installation for compiling the Step")
+	}
+
+	stepDir, err := ioutil.TempDir("", "")
+	require.NoError(b, err)
+
+	defer func() {
+		err := os.RemoveAll(stepDir)
+		require.NoError(b, err)
+	}()
+
+	outputDir, err := ioutil.TempDir("", "")
+	require.NoError(b, err)
+
+	defer func() {
+		err := os.RemoveAll(outputDir)
+		require.NoError(b, err)
+	}()
+
+	stepmanCmd := command.New("stepman", "activate",
+		"--collection", "https://github.com/bitrise-io/bitrise-steplib",
+		"--id", "xcode-test",
+		"--version", "2.5.0", "--update",
+		"--path", stepDir)
+	err = stepmanCmd.Run()
+	require.NoError(b, err)
+
+	packageName := "github.com/bitrise-steplib/steps-xcode-test"
+
+	for _, mode := range []string{"on", "auto"} {
```

Comment:
xcode test is already on go modules. Is there a difference between `on` an `auto`?